examples/image_classification/raspbery_pi/classify.py

```diff
@@ -0,0 +1,164 @@
+  row_size = 20  # pixels
+  left_margin = 24  # pixels
+  text_color = (0, 0, 255)  # red
+  font_size = 1
+  font_thickness = 1
+  fps_avg_frame_count = 10
+
+  classification_result_list = []
+
+  def save_result(result: vision.ImageClassifierResult,
+                      unused_output_image: mp.Image, timestamp_ms: int):
+      classification_result_list.append(result)
+
+  # Initialize the image classification model
+  base_options = python.BaseOptions(model_asset_path=model)
+  options = vision.ImageClassifierOptions(base_options=base_options,
+                                          running_mode=vision.RunningMode.LIVE_STREAM,
+                                          max_results=max_results,
+                                          score_threshold=score_threshold,
+                                          result_callback=save_result)
+  classifier = vision.ImageClassifier.create_from_options(options)
+
+  # Continuously capture images from the camera and run inference
+  while cap.isOpened():
+    success, image = cap.read()
+    if not success:
+      sys.exit(
+          'ERROR: Unable to read from webcam. Please verify your webcam settings.'
+      )
+
+    counter += 1
+    image = cv2.flip(image, 1)
+
+    # Convert the image from BGR to RGB as required by the TFLite model.
+    rgb_image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+    mp_image = mp.Image(image_format=mp.ImageFormat.SRGB, data=rgb_image)
+
+    # Run image classifier using the model.
+    classifier.classify_async(mp_image, time.time_ns() // 1_000_000)
+    current_frame = mp_image.numpy_view()
+    current_frame = cv2.cvtColor(current_frame, cv2.COLOR_RGB2BGR)
+
+    # Calculate the FPS
+    if counter % fps_avg_frame_count == 0:
+        end_time = time.time()
+        fps = fps_avg_frame_count / (end_time - start_time)
+        start_time = time.time()
+
+    # Show the FPS
+    fps_text = 'FPS = {:.1f}'.format(fps)
+    text_location = (left_margin, row_size)
+    cv2.putText(current_frame, fps_text, text_location, cv2.FONT_HERSHEY_PLAIN,
+                font_size, text_color, font_thickness)
+
+    if classification_result_list:
+        # Show classification results on the image
+        for idx, category in enumerate(classification_result_list[0].classifications[0].categories):
+            category_name = category.category_name
+            score = round(category.score, 2)
+            result_text = category_name + ' (' + str(score) + ')'
+            text_location = (left_margin, (idx + 2) * row_size)
+            cv2.putText(current_frame, result_text, text_location, cv2.FONT_HERSHEY_PLAIN,
+                        font_size, text_color, font_thickness)
+
+        classification_result_list.clear()
+
+    # Stop the program if the ESC key is pressed.
+    if cv2.waitKey(1) == 27:
+        break
+    cv2.imshow('image_classification', current_frame)
+
+  classifier.close()
+  cap.release()
+  cv2.destroyAllWindows()
+
+
+def main():
```

Comment:
Let's make sure we document all of the available options here since they all have a strong purpose. Maybe a comment about how devs can get their cameraId.

Reply:
Since it's very reliant on platform-dependent methods, we could use the fact that camera IDs are usually indexed sequentially by the OS, starting from 0. This makes it easy if we use OpenCV.

i.e., By trying to open the camera IDs one by one (say 0-9) using the following should work:
```python
available_cameras = []
    
    for i in range(upper_bound):
        cap = cv2.VideoCapture(i)
        if cap is None or not cap.isOpened():
            print('Camera {} not available'.format(i))
        else:
            print('Camera {} is available'.format(i))
            available_cameras.append(i)
        cap.release()
```